packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -877,6 +921,23 @@ class Actions extends StatefulWidget {
     return action;
   }
 
+  static Action<T>? _maybeFindEnabled<T extends Intent>(BuildContext context, T intent) {
+    Action<T>? action;
+    _visitActionsAncestors(context, (InheritedElement element) {
+      final _ActionsMarker actions = element.widget as _ActionsMarker;
+      final Action<T>? result = actions.actions[intent.runtimeType] as Action<T>?;
```

Comment:
Should we have a more verbose exception thrown here if the `Action` isn't an `Action<T>?`, to let them know that the `actions` map is malformed?